src/Controller/FacebookAuthController.php

```diff
@@ -6,10 +6,12 @@
 use Drupal\social_api\Plugin\NetworkManager;
 use Drupal\social_auth\SocialAuthUserManager;
 use Drupal\social_auth_facebook\FacebookAuthManager;
+
```

Comment:
No need for an space.
use statements must be in alphabetical order (please update them)